code/modules/mob/death.dm

```diff
@@ -64,7 +63,9 @@
 		hud_used.healths.icon_state = "health7"
 
 	timeofdeath = world.time
-	if(mind) mind.store_memory("Time of death: [worldtime2text()]", 0)
+	if(mind) 
+		mind.store_memory("Time of death: [worldtime2text()]", 0)
```

Comment:
You can substitute the `if` check here with a `?.`
`mind?.store_memory("Time of death: [worldtime2text()]", 0)`